lib/components/RecentBlocks.js

```diff
@@ -53,10 +56,19 @@ class RecentBlocks extends React.Component {
           ? block.coinbase.join('')
           : block.coinbase;
       });
-      const expandedData = recentBlocks.map(block => ({
-        mainData: pick(block, this.mainData),
-        subData: omit(block, this.mainData)
-      }));
+
+      if (SPV)
+        this.colHeaders = this.colHeaders.filter(
+          word => this.SPVomit.indexOf(word) === -1
```

Comment:
I think this makes it more readable:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes